### ISSUE TITLE
Creating of the Stone is failing

### ISSUE THREAD

**Issue by sergio101** [GsDevKit/GsDevKit_home#247]
When creating the stone, I keep getting:

```
Error on or near line 99 :: devKitCommandLine startstone sdoh34 -N :: devKitCommandLine startstone sdoh34 -N
Error on or near line 116 :: startStone -b -N sdoh34 :: startStone -b -N sdoh34
Error on or near line 148 :: newExtent -s /Users/sergio/Sites/GsDevKit_home/server/stones/sdoh34/product/bin/extent0.seaside.dbf sdoh34 :: newExtent -s /Users/sergio/Sites/GsDevKit_home/server/stones/sdoh34/product/bin/extent0.seaside.dbf sdoh34
Error on or near line 209 :: createStone -f sdoh34 3.4.0 :: createStone -f sdoh34 3.4.0
```

The entire log is here -> https://gist.github.com/sergio101/84d1e86961bf4f513ce3d880dffc25dc

macos 10.14.3
macbook pro

thanks!

**Comment by sergio101** [comment]
using this commit:

commit 775f12e03497804cbb3fab7b0fdc0407268e1c60 (HEAD -> master, origin/master, origin/HEAD)
Merge: 5e99ef0 0471d97
Author: Dale Henrichs <dale.henrichs@gemtalksystems.com>
Date:   Wed Apr 24 11:33:36 2019 -0700

    Merge pull request #246 from GsDevKit/issue_244

    remove extra args from pushd

**Comment by dalehenrich** [comment]
@sergio101, Thanks for providing all of the details ... it turns out that you've run into a [known issues](http://forum.world.st/Problem-starting-stone-reason-File-is-on-NFS-GEMSTONE-DATADIR-extent0-dbf-tp5060764p5060777.html) for running GemStone on MacOS High Sierra (and possibly later). [This line in the log file was the clue](https://gist.github.com/sergio101/84d1e86961bf4f513ce3d880dffc25dc#file-install-log-L340).

According to our internal bug report for 47197 "Unable to start stone on macOS 10.13 High Sierra", the bug has been fixed in 3.5, 3.4.1, 3.3.7 and later, so I recommend that you create your stone using 3.4.3 (see [the product page](https://gemtalksystems.com/products/gs64/) for latest version information ).
